lib/docblockr-worker.js

```diff
@@ -32,9 +33,9 @@ module.exports =
         self.event = event;
         var regex = {
           // Parse Command
-          'parse': /^\s*(\/\*|###)[*!]\s*$/,
+          'parse': /^\s*((\/\*|###||\/\/)[*!]|)\s*$/,
```

Comment:
Is there a reason for the double pipe (`||`)? I would expect the regex to be ` /^\s*((\/\*|###||\/\/)[*!]|)\s*$/`

Reply:
The reason is that I write more js than regex resulting in mistakes...